website/docs/icons/library/index.js

```diff
@@ -58,6 +62,21 @@ export default class Index extends Component {
     }
   }
 
+  // Groups all filtered icons by category
+  get filteredGroupedIcons() {
```

Comment:
non blocker: in theory, if you want, you can get rid of the `filteredIcons()` getter, and move its code directly in this function, as you had before (essentially is like moving the grouping inside the `filteredIcons()` and then renaming the function)

Reply:
Ok thanks, let me look at it...

Reply:
Check also the alphabetical sorting while you're there (missed that 🤦‍♂️)

Reply:
Pushed up more refactors, thanks.

Reply:
Let me know if it looks good now or if you have more suggested improvements.